.github/workflows/linux.yml

```
@@ -154,6 +154,10 @@ jobs:
             install_steps: pytest
             configure: --with-openssl --enable-debug --disable-unity
 
+          - name: openssl disable --libcurl
+            install_steps: skipall
+            configure: --with-openssl --disable-unity --disable-libcurl-option
```

```suggestion
            configure: --with-openssl --disable-libcurl-option
```
I think we can leave unity enable here for speed and no downsides.

But, would it be even better to merge this combination with `!ipv6`?:
```diff
--- i/.github/workflows/linux.yml
+++ w/.github/workflows/linux.yml
@@ -177,8 +177,8 @@ jobs:
             install_steps: skipall
             generate: -DCURL_USE_OPENSSL=ON -DCURL_USE_GSSAPI=ON -DENABLE_DEBUG=ON -DCURL_LTO=ON
 
-          - name: openssl !ipv6
-            configure: --with-openssl --disable-ipv6 --enable-debug --disable-unity
+          - name: openssl !ipv6 !--libcurl
+            configure: --with-openssl --disable-ipv6 --disable-libcurl-option --enable-debug --disable-unity
 
           - name: openssl https-only
             configure: >-
```

---

thanks, we might as well, to save us from adding one!